commit 7b1d2e164a9c0b94d8ddd3b6107d087e54fe0c8d (refs/changes/62/4062/3)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-06-08T20:31:00+00:00 (1 year, 4 months ago)
    
    TOOLS-2042 prune unused sdcnode builds
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/nodeconfigs.json b/nodeconfigs.json
index eb9e6b7..b012cd5 100644
--- a/nodeconfigs.json
+++ b/nodeconfigs.json
@@ -142,43 +142,6 @@
         "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
-
-    {
-        "// image": "sdc-minimal-multiarch-lts 15.4.1",
-        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
-        "version": "v4.8.5",
-        "patch_files": [
-            "patches/tls-reload-certs-v4.x.patch"
-        ],
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "gcc_version_prefix": "4.9.",
-        "rpath": "$ORIGIN/../lib",
-        "include_gcc_runtime_libs": true,
-        "build_tag": "gz"
-    },
-    {
-        "// image": "sdc-minimal-multiarch-lts 15.4.1",
-        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
-        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
-        "version": "v4.8.5",
-        "patch_files": [
-            "patches/tls-reload-certs-v4.x.patch"
-        ],
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "build_tag": "zone"
-    },
-    {
-        "// image": "sdc-minimal-multiarch-lts 15.4.1",
-        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
-        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
-        "version": "v4.8.5",
-        "patch_files": [
-            "patches/tls-reload-certs-v4.x.patch"
-        ],
-        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
-        "build_tag": "zone64"
-    },
-
     {
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
@@ -203,32 +166,6 @@
         "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
-
-    {
-        "// image": "sdc-minimal-multiarch-lts 15.4.1",
-        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
-        "version": "v4.6.0",
-        "patch_files": [
-            "patches/tls-reload-certs-v4.x.patch"
-        ],
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "gcc_version_prefix": "4.9.",
-        "rpath": "$ORIGIN/../lib",
-        "include_gcc_runtime_libs": true,
-        "build_tag": "gz"
-    },
-    {
-        "// image": "sdc-minimal-multiarch-lts 15.4.1",
-        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
-        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
-        "version": "v4.6.0",
-        "patch_files": [
-            "patches/tls-reload-certs-v4.x.patch"
-        ],
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "build_tag": "zone"
-    },
-
     {
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
@@ -237,66 +174,6 @@
         "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
-    {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.12.17",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "build_tag": "zone"
-    },
-    {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.12.17",
-        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
-        "build_tag": "zone64"
-    },
-    {
-        "// image": "sdc-smartos 1.6.3",
-        "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
-        "version": "v0.12.17",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zone"
-    },
-    {
-        "// image": "sdc-smartos 1.6.3",
-        "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
-        "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
-        "version": "v0.12.17",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
-        "build_tag": "gz"
-    },
-    {
-        "// image": "sdc-base@14.2.0",
-        "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
-        "version": "v0.12.17",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zone"
-    },
-
-    {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.12.14",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "build_tag": "zone"
-    },
-    {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.12.14",
-        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
-        "build_tag": "zone64"
-    },
     {
         "// image": "sdc-smartos 1.6.3",
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
@@ -315,16 +192,6 @@
         "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
-    {
-        "// image": "sdc-base@14.2.0",
-        "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
-        "version": "v0.12.14",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zone"
-    },
-
     {
         "// image": "sdc-multiarch 13.3.1",
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
@@ -359,16 +226,6 @@
         "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
-    {
-        "// image": "sdc-base@14.2.0",
-        "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
-        "version": "v0.12.9",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zone"
-    },
-
     {
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
@@ -430,18 +287,6 @@
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
-    {
-        "// image": "sdc-base@14.2.0",
-        "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
-        "comment": "For use by services running in zones. Plus the DOCKER-515 ECDH cipher support for node 0.10.",
-        "version": "v0.10.48",
-        "patch_files": [
-            "patches/ecdh-ciphers-support-v0.10.patch"
-        ],
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zoneecdh"
-    },
     {
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
@@ -450,32 +295,6 @@
         "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
         "build_tag": "zone"
     },
-
-    {
-        "// image": "sdc-minimal-multiarch-lts 15.4.1",
-        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.10.43",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "build_tag": "zone"
-    },
-
-    {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.10.42",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "build_tag": "zone"
-    },
-    {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.10.42",
-        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
-        "build_tag": "zone64"
-    },
     {
         "// image": "sdc-smartos 1.6.3",
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
@@ -494,44 +313,6 @@
         "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
-    {
-        "// image": "sdc-base@14.2.0",
-        "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
-        "version": "v0.10.42",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zone"
-    },
-    {
-        "// image": "sdc-base@14.2.0",
-        "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
-        "comment": "For use by services running in zones. Plus the DOCKER-515 ECDH cipher support for node 0.10.",
-        "version": "v0.10.42",
-        "patch_files": [
-            "patches/ecdh-ciphers-support-v0.10.patch"
-        ],
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zoneecdh"
-    },
-
-        {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.10.40",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "build_tag": "zone"
-    },
-    {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.10.40",
-        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
-        "build_tag": "zone64"
-    },
     {
         "// image": "sdc-smartos 1.6.3",
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
@@ -550,28 +331,6 @@
         "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
-    {
-        "// image": "sdc-base@14.2.0",
-        "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
-        "version": "v0.10.40",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zone"
-    },
-    {
-        "// image": "sdc-base@14.2.0",
-        "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
-        "comment": "For use by services running in zones. Plus the DOCKER-515 ECDH cipher support for node 0.10.",
-        "version": "v0.10.40",
-        "patch_files": [
-            "patches/ecdh-ciphers-support-v0.10.patch"
-        ],
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zoneecdh"
-    },
-
     {
         "// image": "sdc-base@14.2.0",
         "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
@@ -590,23 +349,6 @@
         "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
         "build_tag": "zone"
     },
-
-    {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.10.30",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "build_tag": "zone"
-    },
-    {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.10.30",
-        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
-        "build_tag": "zone64"
-    },
     {
         "// image": "sdc-smartos 1.6.3",
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
@@ -625,51 +367,6 @@
         "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
-
-    {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.10.29",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "build_tag": "zone"
-    },
-    {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.10.29",
-        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
-        "build_tag": "zone64"
-    },
-    {
-        "// image": "sdc-base@14.2.0",
-        "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
-        "version": "v0.10.29",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zone"
-    },
-    {
-        "// image": "sdc-smartos 1.6.3",
-        "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
-        "version": "v0.10.29",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zone"
-    },
-    {
-        "// image": "sdc-smartos 1.6.3",
-        "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
-        "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
-        "version": "v0.10.29",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
-        "build_tag": "gz"
-    },
-
     {
         "// image": "sdc-multiarch 13.3.1",
         "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
@@ -686,42 +383,6 @@
         "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
-    {
-        "// image": "sdc-smartos 1.6.3",
-        "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
-        "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
-        "version": "v0.10.26",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
-        "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
-        "build_tag": "gz"
-    },
-    {
-        "// image": "sdc-smartos 1.6.3",
-        "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
-        "version": "v0.10.26",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zone"
-    },
-
-    {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.10.25",
-        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
-        "build_tag": "zone64"
-    },
-    {
-        "// image": "sdc-base@14.2.0",
-        "image": "de411e86-548d-11e4-a4b7-3bb60478632a",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
-        "version": "v0.10.25",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zone"
-    },
     {
         "// image": "sdc-smartos 1.6.3",
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
@@ -740,26 +401,6 @@
         "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz"
     },
-
-    {
-        "// image": "sdc-multiarch 13.3.1",
-        "image": "b4bdc598-8939-11e3-bea4-8341f6861379",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib?",
-        "version": "v0.10.24",
-        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
-        "build_tag": "zone64",
-        "// users": "latest"
-    },
-    {
-        "// image": "sdc-smartos 1.6.3",
-        "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
-        "version": "v0.10.24",
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zone"
-    },
-
     {
         "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
         "comment": "For use by agents in the GZ. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
@@ -773,20 +414,5 @@
         "FUTURE_configure_flags": "--without-snapshot --with-dtrace --openssl-use-sys --openssl-libpath=/lib --openssl-includes=??? --shared-zlib --shared-zlib-libpath=/lib --shared-zlib-includes=???",
         "build_tag": "gz",
         "// users": "sdc-amon"
-    },
-    {
-        "// image": "sdc-smartos 1.6.3",
-        "image": "fd2cc906-8938-11e3-beab-4359c665ac99",
-        "comment": "For use by services running in zones. Dynamically link to pkgsrc's openssl and zlib.",
-        "version": "v0.8.26",
-        "npm_version": "v1.4.7",
-        "patch_files": [
-            "patches/node-5feedc5-socket-getFD.patch",
-            "patches/node-reverse-2c41a80-crypto-check-key-type.patch"
-        ],
-        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace --openssl-use-sys --openssl-includes=/opt/local/include --openssl-libpath=/opt/local/lib --shared-zlib --shared-zlib-includes=/opt/local/include --shared-zlib-libpath=/opt/local/lib",
-        "build_env": "LDFLAGS=\"-L/opt/local/lib -R/opt/local/lib\"",
-        "build_tag": "zone",
-        "// users": "manta-mako"
     }
 ]
